v3/integrations/nrgrpc/nrgrpc_server.go

```diff
@@ -306,8 +307,9 @@ func UnaryServerInterceptor(app *newrelic.Application, options ...HandlerOption)
 		option(localHandlerMap)
 	}
 
-	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp interface{}, err error) {
+	return func(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp any, err error) {
 		txn := startTransaction(ctx, app, info.FullMethod)
+		newrelic.GetSecurityAgentInterface().SendEvent("GRPC", req)
```

Comment:
Are we still following the naming convention for gRPC? 
"GRPC" -> "gRPC"

Reply:
That's what the security agent expects (this is an internal protocol between the agents. We can't just change it on our end.